app/bundles/EmailBundle/Config/services.php

```diff
@@ -36,6 +36,9 @@
         ->exclude('../{'.implode(',', array_merge(MauticCoreExtension::DEFAULT_EXCLUDES, $excludes)).'}');
 
     $services->load('Mautic\\EmailBundle\\Entity\\', '../Entity/*Repository.php');
+
+    $services->load('Mautic\\EmailBundle\\Templating\\', '../Entity/*Repository.php');
```

Comment:
What is the reason for this? Making 2 namespaces for 1 class?

Reply:
@escopecz Good catch! There’s no need for two of them. I’ll remove the extra one now